packages/client/src/runtime/DataProxyEngine.ts

```diff
@@ -0,0 +1,243 @@
+import {
```

Comment:
Could we move this file to `@prisma/engine-core` into a new `data-proxy` folder?